<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Passoword</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Passoword</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernamelineedit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passowordlineedit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="retypelineeidt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="DateOFBirthlineedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Monthlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
  </widget>
  <widget class="QLabel" name="daylabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearlabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>year</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Monthcombobox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Jan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feb</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jun</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jul</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aug</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sep</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oct</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nov</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dec</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="daylineedit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearlineedit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Usernameerror">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Username error&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="NotMatchingError">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Not matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageError">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gendergroupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="maleradiobutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="femaleradiobutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AccountTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type:</string>
   </property>
   <widget class="QRadioButton" name="UserRadioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdminRadioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FaviroteGenresGroupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="ActioncheckBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Comdeycheckbox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Romancecheckbox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Dramacheckbox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Horrorcheckbox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="othercheckbox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="registerbutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="registererrorlabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
